pyproject.toml

```diff
@@ -51,6 +51,8 @@ moderngl-window = "^2.3.0"
 mapbox-earcut = "^0.12.10"
 cloup = "^0.7.0"
 requests = "*"
+pytest-clarity = "^0.3.0-alpha.0"
+mypy = "^0.812"
```

Comment:
What do these new dependencies do?

Reply:
pytest-clarity = "^0.3.0-alpha.0" is a mistake, I'll remove it. Mypy is for static type checking. Also, they are both dev.

Reply:
why is it in user dependency lists?

Reply:
strange, I'll fix that.